_topic_map.yml

```diff
@@ -206,28 +206,31 @@ Topics:
 - Name: Installing on bare metal
   Dir: installing_bare_metal
   Topics:
-  - Name: Installing a cluster on bare metal
-    File: installing-bare-metal
-  - Name: Installing a cluster on bare metal with network customizations
-    File: installing-bare-metal-network-customizations
-  - Name: Restricted network bare metal installation
-    File: installing-restricted-networks-bare-metal
-- Name: Deploying installer-provisioned clusters on bare metal
-  Dir: installing_bare_metal_ipi
-  Distros: openshift-origin,openshift-enterprise
-  Topics:
-  - Name: Overview
-    File: ipi-install-overview
-  - Name: Prerequisites
-    File: ipi-install-prerequisites
-  - Name: Setting up the environment for an OpenShift installation
-    File: ipi-install-installation-workflow
-  - Name: Post-installation configuration
-    File: ipi-install-post-installation-configuration
-  - Name: Expanding the cluster
-    File: ipi-install-expanding-the-cluster
-  - Name: Troubleshooting
-    File: ipi-install-troubleshooting
+  - Name: Installing user-provisioned clusters on bare metal
+    Dir: installing_bare_metal_upi
```

Comment:
Unless engineering's changed their minds, we can't actually say "upi" or "ipi" anywhere in the collection, even in URLs. That's why we have filenames like "installing-gcp-user-infra.adoc."

Reply:
They got removed from the docs, but not the URLs.